ebscopy/ebscopy.py

```diff
@@ -50,13 +50,14 @@ def _parse_highlight(text):
 	:key: end_pos: integer representing the end of highlighting in ``clean``
 	
 	"""
+	output = {} #this is OM addition
 	output["orig"]							= text
 	output["clean"]							= ""
 	output["start_pos"]						= 0
 	output["end_pos"]						= 0
 
 	start_tag								= "&lt;highlight&gt;"
-	end_tag									= "&lt;\/highlight&gt;"
+	end_tag									= "&lt;/highlight&gt;"
```

Comment:
Is the linting suggesting this?

Reply:
Yes - with the backslash as it was, pylint gave this error: Anomalous backslash in string: '\/'. String constant might be missing an r prefix.

Reply:
Might have to define this more intelligently. I assume I did this so I could use it multiple times later on.